Simprints/Android-Simprints-ID
# Firestore migration fix realm migrate

Fixed migration code. 

To test: 
1) build and install Simprints ID from master
2) use old demo app with `https://chart.googleapis.com/chart?cht=qr&chl=17a87a5c-105e-4076-8c9e-2c2f17a1955e&chs=160x160&chld=L|0`
3) enrol someone
4) build and install SimprintsID from firestore_migration_fix_realm_migrate (ask Fabio for signing info)
5) verify the same person enrolled before
6) login SimprintsID v2 with `https://simprints.slack.com/files/U4APJNFMY/FAGGTNT9U/image.png`
8) successful verify the same person enrolled before
